op-batcher/batcher/driver.go

```diff
@@ -339,8 +346,11 @@ func (l *BatchSubmitter) loop() {
 	defer ticker.Stop()
 
 	publishAndWait := func() {
-		l.publishStateToL1(queue, receiptsCh)
+		l.publishStateToL1(queue, receiptsCh, daGroup)
 		if !l.Txmgr.IsClosed() {
+			l.Log.Info("Wait for pure DA writes, not L1 txs")
```

Comment:
small nit: might be better to have conditional logic here with the logging based on whether alt da is on or not

Reply:
[2efafab](https://github.com/ethereum-optimism/optimism/pull/11698/commits/2efafab0ad0e0208d7fe853fd840259f24c4e856)

@sebastianst I just copied the logic of the quque's waitgroup, which is also [throwing away its error](https://github.com/ethereum-optimism/optimism/blob/42085ce945bc727707b9d51c8e90e142b05f721e/op-service/txmgr/queue.go#L51). Do you prefer we catch them and log something?

Reply:
[3d31b9a](https://github.com/ethereum-optimism/optimism/pull/11698/commits/3d31b9a5d0fafc414d55252fe2823ee33dcf64ce)